app/build.gradle

```diff
@@ -88,14 +88,6 @@ android {
         }
     }
 
-    variantFilter { variant ->
-        def flavors = variant.flavors*.name
-        // We only need a gecko debug build for now.
-        if (flavors.contains("gecko") && (variant.buildType.name != "debug" || flavors.contains("klar"))) {
-            setIgnore(true)
-        }
-    }
-
```

Comment:
Yeah, I wanted to at least get beta. I changed it to just include that with the debug flavor.